archimedes/archimedes/functions/matrix/__init__.py

```diff
@@ -1,4 +1,5 @@
 from pandas import Series
+import pandas as pd
```

Comment:
These Archimedes imports should all be removed once the demo UMAP workflow is replaced.